DOCPLAN-169: Updated short descriptions for DITA compliance (#109371)
openshift/openshift-docs: virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc

toc::[]

[role="_abstract"]
You can connect a VM to an OVN-Kubernetes custom secondary overlay network. A layer 2 topology connects workloads by a cluster-wide logical switch. The OVN-Kubernetes Container Network Interface (CNI) plugin uses the Geneve (Generic Network Virtualization Encapsulation) protocol to create an overlay network between nodes. You can use this overlay network to connect VMs on different nodes, without having to configure any additional physical networking infrastructure.

ifndef::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
[NOTE]
====
An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/secondary_networks/configuring-multi-network-policy.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You must use the `ipBlock` attribute to define network policy ingress and egress rules for specific CIDR blocks. You cannot use pod or namespace selectors for virtualization workloads.
====
endif::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]

> To configure an OVN-Kubernetes layer 2 secondary network and attach a VM to that network, perform the following steps:

Reviewer: ```suggestion
You can configure an OVN-Kubernetes layer 2 secondary network and attach a VM to it:
```
Avoids self-reference
